semeio/jobs/scripts/gendata_rft.py

```diff
@@ -82,6 +82,18 @@ def _build_parser():
 zone definition. If a k value does not exist the program will stop. The zone
 mapping will be validated and if it is invalid, the pressure is set to -1 and the
 inactive value to 0
+""",
+    )
+    parser.add_argument(
+        "-c",
+        "--csvfile",
+        required=False,
+        type=str,
+        default="gendata_rft.csv",
```

Comment:
Same as above, should this be defaulted to ON?

Reply:
Given the statement in the above comment from author that it should be on by default - should it both output all the separate files and an aggregated version by default? I would also find it to be more sensible to not do it by default. And in the way it is setup now it is pretty much impossible to turn it off, as the `type=str` will fail if one provides a `None` argument..

Reply:
Keeping this as is, to be in sync with JOB_DESCRIPTION